### DISCUSSION TITLE
Cannot add a repo to a list(cannot create a list)

### DISCUSSION THREAD

**Discussion by c0dz** [community/community#57178] [Mobile]
"+ADD TO LIST" feature has a bug in my opinion.
The message (error) in this screenshot contains my previous username. Perhaps it searches for the lists that are related to my previous username.
![20230604_233904.png](https://github.com/community/community/assets/16648282/d6d028b8-2a59-4100-a785-1d03708e5a80)

**Comment by stevepopovich** [comment]
Hi @c0d3z thanks for writing in. Sorry this is happening. We would love to get some more info:

- What are the steps that cause this? Is this when the bottom sheet opens or is it when you change a list?
- Is this consistent for you? 
- Does restarting the app help?

**Reply by c0dz** [reply]
1. after adding the repository to my starred repositories, the "add to list" button comes up. and when I want to add it using the "add to list" button, i get this message
2. yes this happens everytime. for every repository
3. No. restarting the app doesn't help.

**Reply by stevepopovich** [reply]
Gotcha. Sounds like an issue then. As a workaround, would you try logging in and out please? 

Either way, this shouldn't be happening and we will look into this
